build.gradle

```diff
@@ -63,8 +64,16 @@ dependencies {
 
   signature 'org.codehaus.mojo.signature:java17:1.0@signature'
 
-  testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
+  testFixturesImplementation 'org.jenkins-ci.main:jenkins-test-harness:2.44'
```

Comment:
I have seen after my original pull request that "jenkins-test-harness" is also provided vai "org.jenkins-ci.jpi". Not in the latest version but they try to use a version that is compatible with the used Jenkins. Therefore I suggest to remove line 67 and 69.